tests/metrics/convertors.py

```diff
@@ -0,0 +1,194 @@
+import numpy as np
+import pytest
+import torch
+import torch.distributed as dist
+
+import tests.base.utils as tutils
+from pytorch_lightning.metrics.convertors import _apply_to_inputs, _apply_to_outputs, \
+    _convert_to_tensor, _convert_to_numpy, _numpy_metric_conversion, \
+    _tensor_metric_conversion, _sync_ddp, tensor_metric, numpy_metric
+
+
+def test_apply_to_inputs():
```

Comment:
consider to rewrite it as a parametrized test so it is easier to spot failer as it will we just for one configuration
see for example here:
https://github.com/PyTorchLightning/pytorch-lightning/blob/1576ad9963402c1a6f6353df89386d8fe66f5dac/tests/trainer/test_trainer.py#L252-L264

Reply:
Done